api/src/main/java/eu/coatrack/api/ApiKey.java

```diff
@@ -152,4 +153,14 @@ public String toString() {
                 + ", user=" + user
                 + '}';
     }
+
+    public HashedApiKey convertToHashedApiKey() {
```

Comment:
I would suggest to put this into a service class in admin, because:

- the POJO entity "ApiKey" should not include "business logic" 
- as we discussed before, the hash could be salted with the gateway ID, which is assigned by admin

Reply:
I think the service should be a static function in the API folder as both admin and the proxy tests would be using it.